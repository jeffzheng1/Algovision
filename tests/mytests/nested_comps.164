def iter() {
    def i = 0
    lambda() {
        if (i < 10) {
            i = i + 1
            i
        } else {
            null
        }
    }
}

print [i * 2 for i in [i * 2 for i in iter()]]
