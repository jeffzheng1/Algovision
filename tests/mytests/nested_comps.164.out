{0: 4, 1: 8, 2: 12, 3: 16, 4: 20, 5: 24, 6: 28, 7: 32, 8: 36, 9: 40}
